src/mono/mono/mini/mini.h

```diff
@@ -632,6 +632,8 @@ typedef enum {
 	LLVMArgInFPReg,
 	/* Valuetype passed in 1-2 consecutive register */
 	LLVMArgVtypeInReg,
+	/* Pass Vector128 in SIMD registers */
+	LLVMArgVtypeInSIMDReg,
```

Comment:
This passes all simd types not just Vector128.

Reply:
Right, will update the comment. But so far, I only enabled it for `Vector128` in this PR. Will enable other vector types in future PR.